proxy/src/main/java/com/velocitypowered/proxy/protocol/packet/AvailableCommandsPacket.java

```diff
@@ -65,6 +65,7 @@ public class AvailableCommandsPacket implements MinecraftPacket {
   private static final byte FLAG_EXECUTABLE = 0x04;
   private static final byte FLAG_IS_REDIRECT = 0x08;
   private static final byte FLAG_HAS_SUGGESTIONS = 0x10;
+  private static final byte FLAG_IS_RESTRICTED = 0x20;
```

Comment:
This may be out of scope for the PR, but it would be really nice if this was exposed in some kind of way. For example, an api type that wraps around a predicate that indicates the command should be restricted.